style.css

```diff
@@ -1,564 +1,566 @@
 * {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-  }
-  
-  html {
-    font-size: 62.5%;
-  }
-  
-  body {
-    font-family: 'Lato', sans-serif;
-    overflow-x: hidden;
-  }
-  
-  :root {
-    --main-color: #ec5242;
-  }
-  
-  /* ***************** */
-  
-  /* **** NAVBAR ***** */
-  
-  /* ***************** */
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  overflow-x: hidden;
+}
+
+:root {
+  --main-color: #ec5242;
+}
+
+/* ***************** */
+
+/* **** NAVBAR ***** */
+
+/* ***************** */
+.header-nav {
+  display: none;
+}
+
+.icon {
+  display: none;
+  height: 6.4rem;
+}
+
+.container {
+  padding: 2.5rem;
+}
```

Comment:
- [ ] **[OPTIONAL]** Congratulations on successfully adding the space between the logo and your navigation links! You've done a fantastic job. I would like to offer **a small optional suggestion** that could further enhance the appearance of your project. Perhaps you could consider increasing the `font size` by a little bit, say to `14px`. **This would contribute to an even better overall design**. 🤓

  ### Your design:
  ![image](https://github.com/kkmanuu/Capstone-1/assets/107952724/43d2285a-e9ff-4605-89ea-1a22819708a5)